app/notify_client/email_branding_client.py

```diff
@@ -7,8 +7,8 @@ def get_email_branding(self, branding_id):
         return self.get(url="/email-branding/{}".format(branding_id))
 
     @cache.set("email_branding")
```

Comment:
Not totally clear on how this caching works, but if we change this to the following it seems to work properly:
```suggestion
    @cache.set("email_branding-{organisation_id}")
```